src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background: black;
+  margin: 0;
+  padding: 0;
+}
+
+.blue-border {
+  width: 328px;
+  padding: 10px;
+  margin: 20px 20px 40px;
+  background: rgb(48, 141, 192);
+  border: 1px solid rgb(231, 85, 81);
+}
+
+.yellow-border {
+  background: rgb(255, 165, 0);
+  padding: 10px;
+  border: 1px solid rgb(231, 85, 81);
+}
+
+.text-inside {
+  background: rgb(221, 221, 221);
+  /* padding: 10px; */
+}
+
+p {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice